pkg/log/zap/flags.go

```diff
@@ -41,6 +41,10 @@ var levelStrings = map[string]zapcore.Level{
 	"fatal":  zap.FatalLevel,
 }
 
+type encoderConfigFunc func(*zapcore.EncoderConfig)
+
+var ecfs []encoderConfigFunc
```

Comment:
Is this global necessary? If so, can you add an explanatory comment as to why one is needed?

Reply:
Moved `var ecfs []encoderConfigFunc`